kclvm/tools/src/format/tests.rs

```diff
@@ -45,6 +45,10 @@ fn read_data(data_name: &str) -> (String, String) {
 fn test_format_source() {
     for case in TEST_CASES {
         let (data_input, data_output) = read_data(case);
+
+        #[cfg(target_os = "windows")]
+        let data_input = data_input.replace("\n", "\r\n");
```

Comment:
Fix the place where the data generates the newline character instead of replacing it here.

Reply:
I think KCL, like Rust, should have used the LF ending no matter what platform it runs on. CRLF is an anachronism that needs to die. So, I adapted the test cases so that they would run on windows without changing the KCL code.